praw/reddit.py

```diff
@@ -367,11 +367,13 @@ def get(self, path, params=None):
         data = self.request('GET', path, params=params)
         return self._objector.objectify(data)
 
-    def info(self, fullnames):
+    def info(self, fullnames=None, url=None):
         """Fetch information about each item in ``fullnames``.
 
         :param fullnames: A list of fullnames for a comment, submission, or
             subreddit.
+        :param url: A list of urls to retrieve lists of link submissions
+            from.
```

Comment:
Whoops, you're right. Also, do you know why coveralls is having an issue with my AttributeError? I must be missing something.